pkg/datasource/service.go

```diff
@@ -225,6 +225,13 @@ func (s *Service) exportFromVolume(parameters map[string]string) error {
 			parameters[types.DataSourceTypeExportFromVolumeParameterVolumeSize], err)
 	}
 
+	var timeout int
+	if timeout, err = strconv.Atoi(parameters[types.DataSourceTypeExportFromVolumeParameterFileSyncHTTPClientTimeout]); err != nil {
+		s.log.Warnf("DataSource Service: Failed to parse string %v to an valie timeout integer, will ignore this input parameter: %v",
+			parameters[types.DataSourceTypeExportFromVolumeParameterFileSyncHTTPClientTimeout], err)
+		timeout = 60
+	}
```

Comment:
Have we changed the longhorn-manager code accordingly? Can we make the timeout mandatory instead of having a default value? So it can be consistent with the longhorn-manager setting?

[/controller/backing_image_data_source_controller.go#L651-L659](https://github.com/longhorn/longhorn-manager/blob/16c09fdd296bdd893ceaaa2717456d8ea8cc7036/controller/backing_image_data_source_controller.go#L651-L659)
```go
	cmd := []string{
		"backing-image-manager", "--debug",
		"data-source",
		"--listen", fmt.Sprintf("%s:%d", "0.0.0.0", engineapi.BackingImageDataSourceDefaultPort),
		"--sync-listen", fmt.Sprintf("%s:%d", "0.0.0.0", engineapi.BackingImageSyncServerDefaultPort),
		"--name", bids.Name,
		"--uuid", bids.Spec.UUID,
		"--source-type", string(bids.Spec.SourceType),
	}
```